lua/nvim-treesitter/parsers.lua

```diff
@@ -140,6 +140,13 @@ list.bash = {
   maintainers = {"@TravonteD"},
 }
 
+list.fish = {
+  install_info = {
+    url = "https://github.com/krnik/tree-sitter-fish",
+    files = { "src/parser.c", "src/scanner.c" },
+  },
+}
+
```

Comment:
If you want to maintain the queries for fish, you should add your name here in the `maintainers` key (see other entries in the parser list for examples).

Reply:
Will do! @krnik Should I add your name as well?

Reply:
I'll be glad to help if something breaks so yes, please add me too.